### ISSUE TITLE
The losses were low at the beginning of the training

### ISSUE THREAD

**Issue by s775980299** [WangWenhao0716/Adapted-Center-and-Scale-Prediction#13]
I retrained your code. Why does the loss fluctuate from around 0.

**Comment by WangWenhao0716** [comment]
What about the accuracy? I think the loaded pre-trained model is False.

**Comment by s775980299** [comment]
I have set the pre-training model to False, but the training loss value is still even 0.Because I only have one GPU and the batchsize can only be set to 1 for training, so my accuracy is not as good as yours, 
 your loss value is also from 0 when training？

**Comment by WangWenhao0716** [comment]
I see. Please give up training this project. Sorry, the minimum requirement has not been achieved.



发自我的iPhone


------------------ Original ------------------

**Comment by s775980299** [comment]
Excuse me, is it because I only have one GPU that I cannot meet the minimum requirements？
Should I give up your code? I've been looking at it for a long time.

**Comment by WangWenhao0716** [comment]
Please refer the discussion of gpu number and batch size in the original paper. I give the suggestion is because this setting may (very possible) not bring you a convergent result.



发自我的iPhone


------------------ Original ------------------
